generate_golden_files.sh

```diff
@@ -1,3 +1,3 @@
-python -m venv venv
+python3 -m venv venv
```

Comment:
Is this only within the context of the script, or does it change any activated venv outside of the script?

Reply:
confirmed yes.